setup/Swix/Microsoft.FSharp.IDE/Microsoft.FSharp.IDE.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
>46 [](start = 24, length = 2)

You can target `net472`. VS now requires 4.7.2

Reply:
This is updated in our `dev16.1` branch, but we have to keep this at net46 in `master` since we need to be able to F5 in dev15.9.

Reply:
I see. OK.